src/components/Homepage.jsx

```diff
@@ -11,23 +12,53 @@ const Homepage = () => {
       .get("https://nc-news-czlb.onrender.com/api/articles")
       .then((response) => {
         setArticles(response.data);
+        console.log(response.data);
       });
   }, []);
```

Comment:
To answer your question about why this would cause an infinite loop if we put the articles state in the dependancy array...useEffect will re-render whenever anything in that array is changed. So effectively, we're making the API call,  updating the state, which then triggers the useEffect to run again.